apps/prairielearn/src/pages/administratorInstitutions/administratorInstitutions.ts

```diff
@@ -11,7 +13,31 @@ router.get(
   '/',
   asyncHandler(async (req, res) => {
     const institutions = await sqldb.queryRows(sql.select_institutions, InstitutionRowSchema);
-    res.send(AdministratorInstitutions({ institutions, resLocals: res.locals }));
+    const availableTimezones = await getAvailableTimezones();
+    res.send(
+      AdministratorInstitutions({ institutions, availableTimezones, resLocals: res.locals }),
+    );
+  }),
+);
+
+router.post(
+  '/',
+  asyncHandler(async (req, res) => {
+    console.log(req.body);
+    if (req.body.__action === 'insert_institution') {
```

Comment:
I'll admit that our naming isn't perfect, but typically we use "Add" as the verb for user-facing things, and "insert" for the actual SQL query. This is not hugely important, so I won't block on it. These are patterns from PrairieTest, which you unfortunately don't have access to yet.

Reply:
Got it. I have updated this to `add_institution` in commit [fb604ae](https://github.com/PrairieLearn/PrairieLearn/pull/9045/commits/fb604aec1a5e82dbf069cab10a5723c80a85a5c8).